### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by garynlfd** [vuvko/fitw2020#2]
Hi, 

I am interested in RFIW and came across your paper.
You mentioned that you replaced SphereFace with ArcFace.
But in the paper, I think the loss function is more like a softmax loss function.
I also viewed your code. In train.py, there is a variable named all_losses and you wrote three kinds of loss but finally used 'softmax'.
Did you use 'arc' in RFIW?
If I misunderstand something, please correct me.
I would be grateful if you could give me some hints or feedbacks

Thanks!

**Comment by vuvko** [comment]
I haven't used margin-based losses for training. `ArcFace` is mentioned as a base model for finetuning and as a baseline.

**Comment by garynlfd** [comment]
Okay, I got it!
Thank you so much!
